tools/shoestring/tests/wizard/test_setup_file_generator.py

```diff
@@ -29,6 +29,7 @@
 NodeSettingsScreen = namedtuple('NodeSettingsScreen', ['domain_name', 'friendly_name', 'api_https', 'metadata_info'])
 SingleValueScreen = namedtuple('SingleValueScreen', ['current_value'])
 VotingScreen = namedtuple('VotingScreen', ['active'])
+BootstrapScreen = namedtuple('BootstrapScreen', ['active', 'bootstrap_path', 'include_node_key'])
```

Comment:
Do you think it is better to follow this order (same as the class)?
['active', 'include_node_key', 'bootstrap_path'])

Reply:
Does not matter unless we want to keep them in alphabetical order.